custom_components/mikrotik_router/coordinator.py

```diff
@@ -532,6 +532,10 @@ def get_capabilities(self):
                 self.support_capsman = False
                 self._wifimodule = "wifi"
                 
+            elif self.minor_fw_version >= 13:
```

Comment:
I think this will cause issues in future.
Should be (major version = 7 and minor >=13) or major version > 7
At least for now, mikrotik keeps changing this concept a lot recently.